ckanext/ontario_theme/templates/internal/snippets/facet_list.html

```diff
@@ -31,7 +31,7 @@ <h2 class="module-heading">
     <nav>
       <ul class="{{ nav_class or 'list-unstyled nav nav-simple nav-facet' }}">
         {% for item in items %}
-          {{ h.snippet('snippets/translate_facets.html', fields=fields, name=name, item=item, scheming_choices=scheming_choices, extras=extras) }}
+          {{ h.snippet('snippets/translate_facets.html', fields=fields, name=name, item=item, scheming_choices=scheming_choices) }}
```

Comment:
Why is `extras=extras` removed?

Reply:
when I created `{{ h.snippet('snippets/translate_facets.html', fields=fields, name=name, item=item, scheming_choices=scheming_choices, extras=extras) }}` in #286 I added in extras when I really didn't have to. I wasn't aware of it's purpose then, but I understand it now, and it's not needed in the line.